addons/l10n_it_edi/wizard/account_move_send_views.xml

```diff
@@ -6,20 +6,23 @@
             <field name="inherit_id" ref="account.account_move_send_form"/>
             <field name="arch" type="xml">
                 <xpath expr="//div[@name='warnings']" position="inside">
-                    <field name="l10n_it_edi_readonly_xml_export" invisible="1"/>
-                    <field name="l10n_it_edi_enable_xml_export" invisible="1"/>
-                    <field name="l10n_it_edi_readonly_send" invisible="1"/>
-                    <field name="l10n_it_edi_enable_send" invisible="1"/>
```

Comment:
better if you can avoid such changes in stable as I don't think there is maybe no reason to move those invisible fields (xpaths, ...)